onchain/rollups/test/foundry/dapp/Application.t.sol

```diff
@@ -195,7 +195,7 @@ contract ApplicationTest is TestBase {
         _validateNotice(notice, proof);
 
         // reverts if notice is incorrect
-        bytes memory falseNotice = abi.encodePacked(bytes4(0xdeaddead));
+        bytes memory falseNotice = abi.encode(bytes4(0xdeaddead));
```

Comment:
These 2 behaviors are different. But the result is the same because they are both false notices.

Reply:
Ooops, you're right. Let me revert this line.

Reply:
Ok. Either way was ok :)